src/pages/projectSettings/tabs/NotificationsTab/getFormSchema.tsx

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable react/no-unescaped-entities */
```

Comment:
[This rule](https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/no-unescaped-entities.md) is meant to prevent errors we shouldn't disable it

Reply:
I must have accidentally missed my usual cleanup step. Fixed.